main/utils/wheel_menu.c

```diff
@@ -528,18 +522,15 @@ menu_t* wheelMenuTouch(menu_t* menu, wheelMenuRenderer_t* renderer, uint16_t ang
         menuItem_t* cur = menu->currentItem->val;
 
         uint8_t min, max;
-        uint8_t* target = NULL;
         if (menuItemHasOptions(cur))
         {
-            min    = 0;
-            max    = cur->numOptions - 1;
-            target = &cur->currentOpt;
+            min = 0;
+            max = cur->numOptions - 1;
         }
         else
         {
-            min    = cur->minSetting;
-            max    = cur->maxSetting;
-            target = &cur->currentSetting;
```

Comment:
@dylwhich this was fine when you wrote it, but I messed it up in c134a816db0ba2444d3bce01c528ea0f4a8f929d by making `currentOpt` and `currentSetting` different values. `currentSetting` was holding larger-than-8-bit values for the screensaver though